tools/introspection/source/introspection_app.cpp

```diff
@@ -228,15 +228,16 @@ void IntrospectionApp::waitForUserInput(int32_t timeoutMs)
     fileDesc.fd = STDIN_FILENO;
     fileDesc.events = POLLIN;
     constexpr size_t nFileDesc = 1u;
-    /// @todo iox-#1692 Wrap kernel calls with posixCall
-    int32_t eventCount = poll(&fileDesc, nFileDesc, timeoutMs);
-
-    // Event detected
-    if ((eventCount == nFileDesc) && (fileDesc.revents == POLLIN))
-    {
-        updateDisplayYX();
-        refreshTerminal();
-    }
+    iox::posix::posixCall(poll)(&fileDesc, nFileDesc, timeoutMs)
```

Comment:
@zmostafa Could you please update the year in the copyright header at the very top of this file?

Reply:
@mossmaurice Done. 
Is that the rule now ? once I touch a file I have to update the year in the header ?

Reply:
@zmostafa Yes, correct.